README.md

```diff
@@ -6,7 +6,24 @@ Sensor measuring interactive embeddable in CODAP and AP.
 ## Development
 
 Run `npm start` to start webpackdevserver.
+Run `npm run start-https` to run https with certs. See `mkcert` in the SSL section below.
+Run `npm run start-self-signed` to run https with self-signed certs.
 
+
+
+## SSL Information:
+First install mkcert, and install a ROOT CA for yourself:
+See: https://github.com/FiloSottile/mkcert#mkcert
+
+```
+  brew install mkcert
+  mkcert -install
+
+  mkcert -cert-file ./certs/localhost.crt \
```

Comment:
In other projects, e.g. [CLUE](https://github.com/concord-consortium/collaborative-learning/blob/master/README.md#serving-clue-from-httpslocalhost) and [CFM](https://github.com/concord-consortium/cloud-file-manager/blob/master/live-server-ssl-config.js), we've started to move towards a model in which the local host certificate resides in a common location in the user's home directory rather than having to generate a separate certificate for each project. I'd recommend adopting that here.

Reply:
@kswenson thanks for the tip.